tools/payments/cmd/disaster/README.md

```diff
@@ -0,0 +1,5 @@
+# Using Disaster
+
+```bash
+go run main.go -v \
+    -k <operator key file> -e <appropriate environment> -pcr2 <appropriate pcr2 value>
```

Comment:
nit can you update the docs before merge

Reply:
good catch. done